structure/0062-quilt-community-collab.md

```diff
@@ -0,0 +1,250 @@
+    * Whether the community intends to be part of the voting process and, if so, documentation on how that community will follow the voting process (or a commitment to figuring that out before the community may vote on anything)
+2. Once the above information has been provided, a Manager informs all Program members on the Discord server, and opens a discussion thread to figure out how the Program members feel about the new community, and to discuss anything relevant to the application
+3. A Discord channel is set up to allow the administrators and moderators from the new community (who are to be invited and given the Invitee role) to talk with existing Program members regarding their application
+4. Once the voting-eligible Program members have informally agreed that enough information has been gathered, the vote may be created (with options to approve or deny the vote) --- it must be done in a channel the Invitees don't have access to, and follow the voting process outlined earlier in this document
+5. Once the vote has concluded, the discussion channel must be archived and kept read-only, so it can be reviewed later on if needed
+
+### Interpreting the vote
+
+**The vote has passed** when more than 75% of the votes cast at the time of closing are positive. Internal discussions can be removed as needed before the new community is given access to the rest of the server. The new community should also be added to any relevant listings.
+
+Otherwise, **the vote has failed.** Enough time should be given to the Invitees to acknowledge the result and leave the server, but Managers are free to kick them if they don't leave in a timely fashion.
+
+## Process: Adding an Individual Collaborator
+
+Anyone who has experience with moderation and who may be able to contribute productively to the Program may apply to join it. As the Program itself largely exists on Discord, all Individual Collaborators must be present on the Program's Discord server.
+
+This process matches the process for adding a community. However, some guidelines on how to evaluate an Individual Collaborator applicant have been provided below.
+
+### Evaluation criteria
+
+Individuals that apply to the program are evaluated based on a number of criteria, which are examined and discussed by the existing Program members during the application. These criteria include, but are not limited to:
+
+* Why the applicant wishes to join the program
+* The applicant's personal views on social issues
+* The applicant's known approaches to moderation, and the communities they've worked with
+* How the applicant has behaved in the communities that the Program members have access to
+* How well-known the applicant is among Program members
+* Whether the applicant has any known, relevant history
+
+## Process: Removing a community
+
+While we hope that it won't ever be necessary, it's important that we have the ability to remove communities from the program when it becomes necessary.
```

Comment:
```suggestion
While we hope that it won't ever be necessary, it's important that we have the ability to remove communities from the program if need be.
```
"when" sounds inevitable, rather like Thanos. I changed the last "necessary" since it was repeated twice in the same sentance.